### ISSUE TITLE
[i18n] Text with carriage return / backspace is not searched in spreadsheet preview

### ISSUE THREAD

**Issue by Adobe-G11N-I18N-L10N-Team** [adobe/aem-sidekick#387]
### Expected Behaviour
Text with carriage return / backspace can be searched in spreadsheet preview.
https://jira.corp.adobe.com/browse/INTL-17

### Actual Behaviour
'No results found' message when user tries to search for text with carriage return / backspace in spreadsheet

#### Steps to Reproduce
1. Install extension
2. Add project to extension
3. Create spreadsheet file and edit it
4. Add 'test' column
5. Add "ไ r" and "ñ b" to column
6. Preview spreadsheet
7. Search for "ไ r" and "ñ b"

#### Platform and Version
Chrome 131.0.6778.86
Sidekick 7.1.2

https://github.com/user-attachments/assets/f1e242dc-1be4-4eef-9b60-42070aba1f26

**Comment by kmurugulla** [comment]
@rofe - this issue seems fixed . Tested in v7.2.0.

**Comment by rofe** [comment]
@kmurugulla true, thanks for testing. @Adobe-G11N-I18N-L10N-Team please verify.

**Comment by Adobe-G11N-I18N-L10N-Team** [comment]
Issue is not fixed, verified on v7.2.0. New issue raised https://github.com/adobe/aem-sidekick/issues/425
![Image](https://github.com/user-attachments/assets/211f67ed-2d59-4e1d-87f7-066918c2411c)
